### PR TITLE
react-flex) Creating new react-flex package

### PR DESCRIPTION
**Pull request by sopranopillow**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Initial commit for react-flex package, currently there's no functionality.

### CODE REVIEW

File: packages/react-flex/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# @fluentui/react-flex
+
+**React Flex components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```
Comment (ecraig12345) [inline packages/react-flex/README.md:3]: Can you update this description? Probably something like "Flexbox abstraction for..."

File: packages/react-flex/src/Flex.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
```
Comment (ecraig12345) [inline packages/react-flex/src/Flex.tsx:1]: If we're following the same layout as the other packages (which I'd recommend), it should be like this:
```
src/
  Flex.ts           // exports * from ./components/flex/index
  index.ts          // exports * from ./Flex
  components/
    Flex/
      Flex.tsx
      Flex.types.ts
      index.ts      // exports * from Flex and Flex.types
```

File: packages/react-flex/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "module": "commonjs",
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strictNullChecks": true,
+    "noImplicitAny": true,
```
Comment (ecraig12345) [inline packages/react-flex/tsconfig.json:15]: Should be like this. Would you mind making the change in the templates too? (scripts/create-package/plop-templates-react/tsconfig.json and scripts/create-package/plop-templates-node/tsconfig.json)
```suggestion
    "strict": true,
```

File: packages/react-flex/tslint.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["@uifabric/tslint-rules"],
```
Comment (ecraig12345) [inline packages/react-flex/tslint.json:2]: Delete this file

File: packages/react-flex/tsconfig.json (hunk 3 above)
Comment (sopranopillow) [reply packages/react-flex/tsconfig.json]: sure!

**Pull request merged**